react/FilePicker/FilePickerBreadcrumb.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+
+import Typography from '../Typography'
+import Icon from '../Icon'
+import RightIcon from '../Icons/Right'
+import useBreakpoints from '../hooks/useBreakpoints'
+
+const useStyles = makeStyles(() => ({
+  icon: {
+    color: 'var(--actionColorActive)',
+    margin: '0 0.25rem'
+  },
+  previousPath: {
+    color: 'var(--actionColorActive)',
+    cursor: 'pointer'
+  }
+}))
+
+const FilePickerBreadcrumb = ({ path, onBreadcrumbClick }) => {
+  const { isMobile } = useBreakpoints()
+  const classes = useStyles()
+  const hasPath = path && path.length > 0
```

Comment:
useMemo ?

Reply:
No, this logic is not very complex

https://blog.logrocket.com/rethinking-hooks-memoization/

Reply:
Je ne pense pas que ça soit pertinent dans ce genre de cas, je suis presque sûr que la mémoïsation sera plus couteuse que ce test.